content/administration/upgrade/odoo_sh.rst

```diff
@@ -1,46 +1,72 @@
 
-=======================
-Odoo.sh version upgrade
-=======================
+=======
+Odoo.sh
+=======
 
-Download and Upload your database
-=================================
+Test upgrade request
+====================
 
-Download a dump of your database (from the :ref:`Builds view <odoosh-gettingstarted-builds-download-dump>`), choose the
-exact copy and without filestore options. Upload the .sql.gz dump on https://upgrade.odoo.com/upload and
-select the Testing Purpose. If you have custom code, you can choose to have it upgraded by us, or do it yourself. Once
-it's processed, you'll get a dump of the database in return.
+Download a dump of your database (from the :ref:`Builds view <odoosh-gettingstarted-builds-download-dump>`),
+choose the **exact copy and without filestore** options. Upload the .sql.gz dump on https://upgrade.odoo.com/upload and
+select the testing purpose. Once it's processed, you'll get a dump of the database in return.
 
-.. warning::
-   Do *not* upload *backups* of your production database (found in the Backups tab of the production branch) as these
-   are incompatible with the Upgrade platform - they contain your complete sources, etc. that are not needed for the
-   upgrade. Make sure to download a **Dump** instead - either through the Backups tab using the *Download Dump* button
-   or through the Builds page by using the *Download Dump* entry of the contextual menu of your latest production build.
+.. important::
+   To have the most up to date test version of your database, create a back-up of your
+   production database just before creating your request. Upload it in your staging branch
+   selecting **Exact dump** and **Without filestore** and then click start.
 
 Test your upgraded database
 ===========================
 
-Create a staging branch that will run the upgraded database. Either make sure your production branch's code is
-compatible between the two Odoo versions and fork your production branch, or make a new staging branch containing
-the upgraded code.
+Create a staging branch that will run the upgraded database. Either make sure your production
+branch's code is compatible between the two Odoo versions and fork your production branch, or make a
+new staging branch containing the upgraded code.
 
-Once the staging build is done (it doesn't matter if it failed due to the version incompatibility), import your
-upgraded dump in the backups tab of the branch. The platform will automatically detect the version of the dump and
-change the version of Odoo's source code to the corresponding version for the build.
+Once the staging build is done (it doesn't matter if it failed due to the version incompatibility),
+import your upgraded dump in the backups tab of the branch. The platform will automatically detect
+the version of the dump and change the version of Odoo's source code to the corresponding version
+for the build.
```

Comment:
- present tense (instead of future)
- this could be a note, since it's an additional info rather than an explanation of what the user must do.

```suggestion
Once the staging build is done (it doesn't matter if it failed due to the version incompatibility),
import your upgraded dump in the backups tab of the branch. 

.. note::
   The platform automatically detects the version of the dump and changes the version of Odoo's
   source code to the corresponding version for the build.
```